spec/views/spotlight/pages/_form.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe 'spotlight/pages/edit', type: :view do
+  let(:exhibit) { stub_model(Spotlight::Exhibit) }
+  let(:page) { stub_model(Spotlight::FeaturePage, exhibit: exhibit) }
+  let :blacklight_config do
+    Blacklight::Configuration.new
+  end
+  before do
+    assign(:page, page)
+    # allow(view).to receive_messages(blacklight_config: blacklight_config)
+    allow(view).to receive_messages(blacklight_config: blacklight_config,
+                                    available_index_fields: [],
+                                    available_view_fields: [],
+                                    featured_images_path: '/foo')
+  end
+  it '' do
```

Comment:
hi @mejackreed! Can you add a test description here?

Reply:
Thanks nice catch. I was too eager :) Added.